### PR TITLE
feat: [ON-2327] Basic+ updates

### PR DESCRIPTION
**Pull request by thehighestprimenumber**

### CODE REVIEW

File: app/src/backend/InventoryProgressService.ts [hunk 1]
```diff
@@ -74,6 +75,11 @@ export default class InventoryProgressService {
               } else if (sector.referenceNumber === "III") {
                 return [1, 3].includes(lastDigit);
                 // return subcategories ending with 1 and 3
+              } else if (
```
Comment (lemilonkh) [inline app/src/backend/InventoryProgressService.ts:78]: Comment seems to be outdated

File: app/src/backend/InventoryProgressService.ts [hunk 2]
```diff
@@ -74,6 +75,11 @@ export default class InventoryProgressService {
...
+                sector.referenceNumber === "IV" ||
+                sector.referenceNumber === "V"
+              ) {
+                return [1].includes(lastDigit);
```
Comment (lemilonkh) [inline app/src/backend/InventoryProgressService.ts:82]: Do we only have scope 1 for sectors 4 and 5?

File: app/src/components/Cards/SectorCard.tsx [hunk 3]
```diff
@@ -115,13 +93,16 @@ export function SectorCard({
                 letterSpacing="wide"
                 className="py-[16px]"
               >
-                <Trans t={t}>scope-required-for-gpc</Trans>:{" "}
-                {sectorScopes?.join(", ") || "none"}
+                {InventoryTypeEnum.GPC_BASIC_PLUS === inventory?.inventoryType
+                  ? t("scope-required-for-gpc+")
+                  : t("scope-required-for-gpc")}
+                {": "}
+                {(sectorScopes || [])?.join(", ") || "none"}
```
Comment (lemilonkh) [inline app/src/components/Cards/SectorCard.tsx:100]: Does the none string here need i18n?

File: app/src/components/HomePage/TabHeader.tsx [hunk 4]
```diff
@@ -60,12 +64,12 @@ export function TabHeader({
         {!isPublic ? (
           <Trans
             i18nKey="gpc-inventory-description"
-            values={{ year: year }}
+            values={{ year: inventory }}
```
Comment (lemilonkh) [inline app/src/components/HomePage/TabHeader.tsx:67]: This seems wrong? Or is there a year number stored in a variable named `inventory`?

File: app/src/util/constants.ts [hunk 5]
```diff
@@ -1,31 +1,103 @@
...
+export const SECTORS: ISector[] = [
   {
-    sectorName: "stationary-energy",
+    referenceNumber: "I",
+    number: 1,
+    name: "stationary-energy",
+    description: "stationary-energy-description",
+    icon: TbBuildingCommunity,
+    inventoryTypes: {
+      [InventoryTypeEnum.GPC_BASIC]: { scopes: [1, 2] },
```
Comment (lemilonkh) [inline app/src/util/constants.ts:50]: Shouldn't this be 1 and 3 for most sectors in basic?

**Review by lemilonkh — APPROVED** [review]
Great work, thanks 🙌✨

File: app/src/app/[lng]/[inventory]/data/[step]/[subsector]/page.tsx [hunk 6]
```diff
@@ -102,6 +106,11 @@ function SubSectorPage({
         return t("II");
       case "3":
         return t("III");
+      case "4":
+        return t("IV");
+      case "5":
+        return t("V");
```
[the remark this answers is not in the document]
Comment (thehighestprimenumber) [reply app/src/app/[lng]/[inventory]/data/[step]/[subsector]/page.tsx:112]: we just have IPPU and AFOLU in the figma designs

File: app/src/util/constants.ts (hunk 5 above)
Comment (thehighestprimenumber) [reply app/src/util/constants.ts:50]: that's for waste

**Pull request merged**